pkgs/applications/office/appflowy/default.nix

```
@@ -1,31 +1,49 @@
-{ stdenv
-, lib
-, fetchzip
-, autoPatchelfHook
-, makeWrapper
-, copyDesktopItems
-, makeDesktopItem
-, gtk3
-, xdg-user-dirs
-, keybinder3
-, libnotify
+{
+  stdenvNoCC,
+  lib,
+  fetchzip,
+  autoPatchelfHook,
+  makeWrapper,
+  copyDesktopItems,
+  makeDesktopItem,
+  gtk3,
+  xdg-user-dirs,
+  keybinder3,
+  libnotify,
 }:
 
-stdenv.mkDerivation rec {
+let
+  dist =
+    {
+      x86_64-linux = {
+        urlSuffix = "linux-x86_64.tar.gz";
+        hash = "sha256-u8Y676sBLDI57z7Zpe72ELaibe83EFwB0oq+ondRIcU=";
+      };
+      x86_64-darwin = {
+        urlSuffix = "macos-universal.zip";
+        hash = "sha256-muTmxNm3dFDeBzYyZZD1GiaHYuy427tqhxz2bL3Umis=";
+      };
+      aarch64-darwin = {
```

> ❓ I dislike that url and hash are duplicated for the darwin platform in dist. Can someone think of a more elegant way to do this?

You could set `aarch64-darwin` to `dist.x86_64-darwin`

```suggestion
      aarch64-darwin = dist.x86_64-darwin;
```

---

Thanks for the suggestion, @jopejoe1, yet that results in `error: infinite recursion encountered` for me. What am I doing wrong?

---

my bad, did not think about that.

changing dist to a recursive attrset (`rec`) should make it possible to set `aarch64-darwin = x86_64-darwin;`

---

Excellent! Thanks, @jopejoe1, I've made and pushed the appropriate changes to this PR.